### DISCUSSION TITLE
Unable to download

### DISCUSSION THREAD

**Discussion by zcrv5** [aclap-dev/video-downloadhelper#530] [Help and Q&A]
### Your question

Some websites can detect the video, but when I click the download will report an error

### Addon & CoApp version and details

{
  "version": "9.0.1.13",
  "target": "google",
  "channel": "stable",
  "page_url": "https://jushuitan.study.moxueyuan.com/new/course/6916733?from=train&type=1&questid=295101&fromParam=courseInTrain&ret=train&resource_relation=%5B%7B%22id%22%3A%22295101%22,%22type%22%3A3%7D%5D",
  "error": {
    "details": "SideDownload error: : [hls @ 000001ba5beea640] Error when loading first segment 'https://vod.moxueyuan.com/6d050cf1vodtransbj1500000054/3026970d1253642696995284043/tsc_12_3_0.ts?sign=ee28f78c034e52e7e0b667d66c248b1e&t=664df213&us=124457-DRDYenfUtO'\r\n[in#0 @ 000001ba5beea500] Error opening input: Invalid data found when processing input\r\nError opening input file https://vod.moxueyuan.com/6d050cf1vodtransbj1500000054/3026970d1253642696995284043/voddrm.token.eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9~eyJ0eXBlIjoiRHJtVG9rZW4iLCJhcHBJZCI6MTUwMDAwMDA1NCwiZmlsZUlkIjoiMTI1MzY0MjY5Njk5NTI4NDA0MyIsImN1cnJlbnRUaW1lU3RhbXAiOjAsImV4cGlyZVRpbWVTdGFtcCI6MjE0NzQ4MzY0NywicmFuZG9tIjowLCJvdmVybGF5S2V5IjoiIiwib3ZlcmxheUl2IjoiIiwiY2lwaGVyZWRPdmVybGF5S2V5IjoiMGVjOGQ2NzM4MTJlMDg5NWVkMmNkOTUzZGYyNDdhMDYyMWU1ZmNjNzI4ZWRiMjU4NjJlODUyMTU1NmM4NjRiZWVjN2ZkZjljOTg1M2Q5NWZmNGUxNzI5Yjc3NDk1YTNiZjU5NDIwZmM1ZmQ0MGRkMTI4MWM3YjU5NGY2MDBkY2Y3YWNmN2Q0ZTRkNzE5Zjg3YTlhZjAzMGViNTE0YjJmZmFlNjQxOTkwOWZjNDRjZTI5ZDk0OGEwMmFiYjU1YzYyNTQwOTg0YzE2MjUzYTI4YTZlMzZkMTk4YzU5ZDc1YjY0YTQzNjM4NDNkZDkxODdmNzlmMWFjOGFlNjBhOWI0MCIsImNpcGhlcmVkT3ZlcmxheUl2IjoiMzE5NTQ5MTUwZjY5MzgwNDE5NzNjZDEzY2Y5MzMwODc4ZmQwYTI4NTRiMDFlY2RkMzE4OTQ3ZmU2OGFjYzYxMDMxYzk4YzQyMzEzZmVjNGE5MWZmMmU4ZDFjZTg3Mjc1MzAzMGUxYjVmZGJjNDVmNDgxNGNhMmRjZDY3MjdjMTdkOTZhOGFjYjk2YmRlNGQ2MGI2NmE1NmQyZDM3MzAwYjRhMDU3YzMzZjcyMTI2NDdjMTE0ZDAxZGY4NjNjNzdjZDAyZGU2Y2I3MTkwNTUwNTcxODI2MDc1MjNjOTE0NjEyMGEzYmEwMTk0MzY4OTFlYjcyNmRkMTJjYjcyZThmMyIsImtleUlkIjoxLCJzdHJpY3RNb2RlIjowLCJwZXJzaXN0ZW50IjoiIiwicmVudGFsRHVyYXRpb24iOjAsImZvcmNlTDFUcmFja1R5cGVzIjpudWxsfQ~7VZBAYFcydEjecsyDypC_7B6BamCJEB3vW5x8mY4AjA.tsc_video_12_3.m3u8?sign=ee28f78c034e52e7e0b667d66c248b1e&t=664df213&us=124457-DRDYenfUtO.\r\nError opening input files: Invalid data found when processing input\r\n",
    "downloadable_id": "downloadable_5627932739056250",
    "error": "coapp_failure",
    "id": "coapp_failure_062652fe-3397-48c1-bfcf-12d39b796fe4"
  }
}

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

I went to the web page reported in the page_url part of your problem report.  It looks like it's all in Japanese (that's a guess) so I have no idea what I'm looking at.  But it looked a lot like a prompt for a login.  If this site requires a registration, I will probably not be able to help you.  I make it a personal policy to not add yet another login registration to my name just to help out a fellow user.

However, I will take a total wild-ass guess.  Take this for what it's worth.  It might help you.  It might be totally irrelevant.  You can't know unless you try it.  Search this forum for discussions dealing with antivirus & firewalls.  You might find some advice that applies to you.  In addition, you should always refer to the Wiki.  There's a link to that at the top of every page of this forum.  The Wiki is a sort of user manual.  You should make sure you read it.  All of it.  There is some commentary in there as well about antivirus & firewalls.
